--- a/plugin/httpserver.py
+++ b/plugin/httpserver.py
@@ -39,7 +39,7 @@ def getAllNetworks():
 	tempaddrs = []
 	# Get all IP networks
 	if fileExists('/proc/net/if_inet6'):
-		if has_ipv6 and version.major >= 12:
+		if has_ipv6:
 			proc = '/proc/net/if_inet6'
 			for line in file(proc).readlines():
 				# Skip localhost
@@ -179,7 +179,7 @@ def HttpdStart(session):
 
 		# start http webserver on configured port
 		try:
-			if has_ipv6 and fileExists('/proc/net/if_inet6') and version.major >= 12:
+			if has_ipv6 and fileExists('/proc/net/if_inet6'):
 				# use ipv6
 				listener.append( reactor.listenTCP(port, site, interface='::') )
 			else:
@@ -234,7 +234,7 @@ def HttpdStart(session):
 				sslroot = AuthResource(session, temproot)
 				sslsite = server.Site(sslroot)
 
-				if has_ipv6 and fileExists('/proc/net/if_inet6') and version.major >= 12:
+				if has_ipv6 and fileExists('/proc/net/if_inet6'):
 					# use ipv6
 					listener.append( reactor.listenSSL(httpsPort, sslsite, context, interface='::') )
 				else:
@@ -253,7 +253,7 @@ def HttpdStart(session):
 		# Streaming requires listening on 127.0.0.1:80
 		if port != 80:
 			try:
-				if has_ipv6 and fileExists('/proc/net/if_inet6') and version.major >= 12:
+				if has_ipv6 and fileExists('/proc/net/if_inet6'):
 					# use ipv6
 					# Dear Twisted devs: Learning English, lesson 1 - interface != address
 					listener.append( reactor.listenTCP(80, site, interface='::1') )
